File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task71_CLAUDE_claude-3-5-sonnet-20240620.java

public class StringToInteger {
    public static void main(String[] args) {
        if (args.length == 0) {
            System.out.println("Please provide a string argument.");
            return;
        }
        
        String input = args[0];
        try {
            int result = Integer.parseInt(input);
            System.out.println("Converted integer: " + result);
        } catch (NumberFormatException e) {
            System.out.println("Invalid input. Cannot convert to integer.");
        }
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task71_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>
#include <string>
#include <stdexcept>

int convertToInt(const std::string& input) {
    try {
        return std::stoi(input);
    } catch (const std::invalid_argument& e) {
        return -1;
    } catch (const std::out_of_range& e) {
        return -1;
    }
}

int main(int argc, char* argv[]) {
    if (argc != 2) {
        std::cout << "Usage: " << argv[0] << " <string>" << std::endl;
        return 1;
    }

    std::string input = argv[1];
    int result = convertToInt(input);

    if (result != -1) {
        std::cout << "Converted integer: " << result << std::endl;
    } else {
        std::cout << "Invalid input: Cannot convert to integer" << std::endl;
    }

    return 0;
}
